### PR TITLE
Creation of new TPC Offline HTML QA module

### PR DESCRIPTION
**Pull request by rosstom2232**
New module that writes html output for the QA histograms produced by TpcClusterQA and TpcChanQA modules
Default is to just write the ClusterQA output for now, but that can be changed if need be

### CODE REVIEW

File: subsystems/tpc/macros/draw_calo.C [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#include <qahtml/OnlProdClient.h>
+#include <qahtml/TPCDraw.h>
+
+R__LOAD_LIBRARY(libonlprodcalo.so)
+
+void draw_calo(const char *rootfile) {
+
+  OnlProdClient *cl = OnlProdClient::instance();
+  /* cl->Verbosity(1); */
+  OnlProdDraw *ex = new TPCDraw();
```
Comment (osbornjd) [inline subsystems/tpc/macros/draw_calo.C:10]: Chris made some changes to rename this to `QADraw` - can you do a git pull and retest to make sure everything still works

File: subsystems/tpc/macros/draw_calo.C [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#include <qahtml/OnlProdClient.h>
+#include <qahtml/TPCDraw.h>
+
+R__LOAD_LIBRARY(libonlprodcalo.so)
+
+void draw_calo(const char *rootfile) {
```
Comment (osbornjd) [inline subsystems/tpc/macros/draw_calo.C:6]: Lets rename this to draw_tpc

File: subsystems/tpc/Makefile.am [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+AM_CPPFLAGS = \
+  -I$(includedir) \
+  -I$(OFFLINE_MAIN)/include \
+  -isystem$(ROOTSYS)/include 
+
+lib_LTLIBRARIES = \
+  libonlprodcalo.la
+
+libonlprodcalo_la_LIBADD = \
```
Comment (osbornjd) [inline subsystems/tpc/Makefile.am:11]: Lets rename this library to something like `qadrawtpc`

**Review by osbornjd — CHANGES_REQUESTED** [review]
We shouldn't need the sPhenixStyle files in there - shouldn't we be able to access them from the nightly build?

Try doing a git pull and making sure that your changes mesh well with some of the renaming Chris did over the weekend

**Comment by rosstom2232** [comment]
Just saw Joe's comments, I'll fix those now!

**Pull request merged**